src/etcd/etcd_utils.h

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
blank line - you could run clang-format over all source files to take care of this

Reply:
clang-format kept this line, weirdly enough. Removed manually